.gitignore

```diff
@@ -1,5 +1,4 @@
 .vs
-.vscode
```

Comment:
Please could you run all 4 of the autogeneration pipeline tasks (see https://dev.azure.com/azure/azure-resource-manager-schemas/_build) against this branch to ensure they run successfully and that the output is as expected.

Reply:
Do you mean `Autogenerate (Batch 0~3)` ? Which branch should i specify?

BTW, as just mentioned, I executed `generate-all` locally, and attached the commit in last comment. All RP generated successfully. Do i still need run the pipeline tasks?

Reply:
Thanks! If you were able to run `generate-all` locally then no need to run the pipelines.